src/config.ts

```diff
@@ -137,3 +129,18 @@ export const URLS = {
     process.env.REACT_APP_PUBLIC_MERKLE_PROOF_HELPER ||
     "https://api.umaproject.org/get-claims",
 };
+
+export const expirationTvl = +(
+  process.env.REACT_APP_PUBLIC_EXPIRATION_TVL ?? 200_000_000
+);
+
+const maxPayout = 2;
+const minPayout = 0.1;
+export const expirationPayout = Math.max(
+  minPayout,
+  Math.min(maxPayout, expirationTvl / 10 ** 9)
+).toFixed(3);
```

Comment:
I've decided to go with env variables because they are more flexible even if less clean, I'm open to change this but I have some questions first

- What would happen if we get the expiration payout from the EMP but then there is a dispute? 
- As far as I understand it, the time at which the synth are redeemable is `EMPExpirationTime` + some other time that depends on the OO liveness, or if there a dispute and things gets escalated to the DVM etc... And I don't know of a way of getting the "right" time, without defaulting to the max which would be 2 days after a settlement without disputes